active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,234 @@
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a simple NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+```
+
+![](images/numberbox-placeholder-text.png)
+
+### Enable calculation support
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for simple calculations" 
+    Header="Enter equation:" 
+    PlaceholderText="A + B * C" 
+    AcceptsCalculation="True" />
+```
+
+![](images/numberbox-acceptscalculation.png)
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for items in order" 
+    StepFrequency="0.1"
+    UpDownButtonsEnabled="True"
+    HyperDragEnabled="True" 
+    HyperScrollEnabled="True" />
+```
+
+![](images/numberbox-updownbuttons.png)
+
+### Format display and output
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for solution components" 
+    Header="Enter a percentage:"
+    PlaceholderText="2/3" 
+    MinValue="0"
+    MaxValue="100" 
+    AreLeadingZerosTrimmed="False"
+    DecimalPrecision="5"
+    DoesInputRound=False" />
+```
+
+## Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+## API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+
+## API Notes
+
+### Notable Properties  
+
+| Name | Description |
+|:-:|:--|
+| StepFrequency | Gets or sets the value part of a value range that steps should be created for. |
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+runtimeclass NumberBoxValueChangedEventArgs
+{
+    String Text;
+    Double Value
+};
+
+runtimeclass NumberBoxTextChangedEventArgs
+{
+    String Text;
+    Double Value
+};
+
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.TextBox
+{
+    NumberBox();
+    
+    Double Value;
+    Boolean AcceptsCalculation;
+    
+    Boolean UpDownButtonsEnabled;
+    Boolean HyperDragEnabled;
+    Boolean HyperScrollEnabled;
+    Double StepFrequency;
+  
+    Boolean AreLeadingZerosTrimmed;
+    Double DecimalPrecision;
+    Boolean DoesInputRound;
+    
+    Double MinValue;
```

Comment:
What happens to the up/down buttons when the min/max is hit. Do they disable? Consider that the buttons are enabled, the step size is 1, the minimum is 0, and the current value is 2. I click Down-->Down, so the value is currently 0. Does the down button get disabled?

Reply:
Disabling is nice but not essential IMHO

Reply:
@mrlacey provided a prototype/analysis earlier and raised a good point that stepping beyond a min or max should coerce the value to the min or max, respectively. E.g., user enters value of 0.5, StepFrequency="0.2", MinValue="0.0", MaxValue="1.0", if the user were to use hyper scrolling || hyper drag ||  UpDownButtons to step, updates would stop at 0.1 or 0.9 rather than at the bounds. 

It would be great to see the buttons disabled when the MinValue/MaxValue are hit, but as @mrlacey points out, it has not been an essential ask.

Reply:
Added to API Notes > Behavioral Components > SpinButton.